cmd/update_test.go

```diff
@@ -0,0 +1,29 @@
+package cmd
+
+// import (
```

Comment:
Can you please clean up this file when it is not supposed to contain any running test?

Reply:
I removed this by myself now.